ctapipe/io/pointing.py

```diff
@@ -0,0 +1,131 @@
+from typing import Any
+
+import astropy.units as u
+import numpy as np
+import tables
+from astropy.time import Time
+from scipy.interpolate import interp1d
+
+from ctapipe.core import Component, traits
+
+from .astropy_helpers import read_table
+
+
+class PointingInterpolator(Component):
+    """
+    Interpolate pointing from a monitoring table to a given timestamp.
+
+    Monitoring table is expected to be stored at ``/dl0/monitoring/telescope/pointing``
+    in the given hdf5 file.
+    """
+
+    bounds_error = traits.Bool(
+        default_value=True,
+        help="If true, raises an exception when trying to extrapolate out of the given table",
+    ).tag(config=True)
+
+    extrapolate = traits.Bool(
+        help="If bounds_error is False, this flag will specify whether values outside"
+        "the available values are filled with nan (False) or extrapolated (True).",
+        default_value=False,
+    ).tag(config=True)
+
+    def __init__(self, h5file=None, **kwargs):
```

Comment:
please document the h5file argument (what is expected, what happens if it's None). E.g. if h5file is specified, PointingInterpolator will look for the tables in [path] and load all that are available for each tel_id.  If it is not specified, the user should call `add_table()` to add tables for relevant tel_ids.